experimental/traffic-portal/package.json

```diff
@@ -61,12 +61,14 @@
     "@angular/platform-server": "^16.0.4",
     "@angular/router": "^16.0.4",
     "@nguniversal/express-engine": "^16.0.2",
+		"@types/node-forge": "^1.3.2",
```

Comment:
this should be a dev dependency

Reply:
also doesn't match the rest of the file's indentation style. Tabs are superior, of course, but `npm` will re-format it to use spaces by default, which gets annoying down the road.